Source/SharpDX.Direct3D12/Mapping.xml

```diff
@@ -238,7 +238,10 @@
     <map method="ID3D12GraphicsCommandList::IASetIndexBuffer" property="false"/>
 
     <map method="ID3D12Device::RegisterBudgetChangeCallback" visibility="internal"/>
-
+    <map method="ID3D12GraphicsCommandList::BeginEvent" visibility="internal" />
+    
+    <map method="ID3D12GraphicsCommandList1::OMSetDepthBounds" visibility="private" />
```

Comment:
Haven't checked in details, but why this one is made private?

Reply:
Because we Have BeginEvent with String instead of IntPtr logic, and we have SetDepthBounds that calls OMSetDepthBounds

Reply:
Could you paste a link where this call `SetDepthBounds` is made?, failing to find it via github...

Reply:
My bad, did not commited the new GraphicsCommandList1.cs, will add it later

Reply:
So if `OMSetDepthBounds` is exposed as-is, we should not introduce a manual method but just add `name="SetDepthBounds"` in the xml `<map method...` and remove `visibility="private"`

Reply:
How to do the remap?

Reply:
` <map method="ID3D12GraphicsCommandList1::OMSetDepthBounds" name="SetDepthsBounds" />`

Reply:
note that there are plenty of examples all around in all the mapping xml files for similar cases/more complex cases

Reply:
Dunno why I didn't even consider that :) my bad, will fix

Reply:
Should be fine now